packages/grafana-ui/src/components/Graph/Graph.tsx

```diff
@@ -27,6 +27,7 @@ export interface GraphProps {
   showLines?: boolean;
   showPoints?: boolean;
   showBars?: boolean;
+  events?: FlotEvents;
```

Comment:
We do not want to expose any internal flot types in the API interface of this Graph component, this should be a DataFrame. We plan to work on Graph component after 7.0, make it take in DataFrame[} directly, then this this events would also not be needed and would be a DataFrame meta or field config property